05_searchmatch_js/style.scss

```diff
@@ -0,0 +1,73 @@
+/* PRESETS*/
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    font-size: 1.6rem;
+}
```

Comment:
Using `*` can be too drastic. You could just target `body` or specify a reset style for each select as you go. Or just use https://meyerweb.com/eric/tools/css/reset/